### ISSUE TITLE
WebDAV problem

### ISSUE THREAD

**Issue by FMuro** [mickstar/Zoo-For-Zotero#77]
I [recently reported](https://forums.zotero.org/discussion/83554/zotero-loop-syncs-2000-items#latest) a endless syncing loop in Zotero, which the Zotero developers told me could be related to the use of a third-party Zotero client. Since the only one I've used is this one (which I find extremely useful), I thought of reporting here. The Zotero developers found a way of fixing this. You can find the fix [here](https://github.com/zotero/zotero/blob/bccf5ff0b2ed5f3afde168c807abf7da9d84be0c/chrome/content/zotero/xpcom/storage/webdav.js#L456-L470). I hope that, if the cause was Zoo, this could help you to improve whatever necessary. Thanks for Zoo!

**Comment by mickstar** [comment]
Thanks for this message. I will find sometime to look into this

**Comment by dbonattoj** [comment]
I just saw this, I guess it is related  :) (?)

![image](https://user-images.githubusercontent.com/28501597/84833526-c26b8800-b02f-11ea-99ae-ab3b2c3532f1.png)

**Comment by FMuro** [comment]
@dbonattoj exactly, that's the way my data base corruption was fixed by the Zotero people, and that's why I opened this bug with a link to the commit where that was implemented.
